src/client.rs

```diff
@@ -132,16 +151,40 @@ impl Client {
                 HeaderValue::from_str(client_id).unwrap(),
             );
         }
-        if let Some(stripe_version) = &self.headers.stripe_version {
+
+        let user_agent: String = format!("Stripe/v3 RustBindings/{}", VERSION);
+
+        if let Some(app_info) = &self.app_info {
+            let formatted: String = format_app_info(app_info);
+            let user_agent_app_info: String = format!("{} {}", user_agent, formatted);
             headers.insert(
-                HeaderName::from_static("stripe-version"),
```

Comment:
Do we not need the `stripe-version` header anymore, or did this accidentally get removed (or moved somewhere else and I just missed it?)

Reply:
whoops! I believe this was taken out in error